methods/check_taxon.nf

```diff
@@ -1,6 +1,6 @@
 def check_taxon(species) {
   "Check if organism name exists"
-  if (species == "streptococcus") {
+  if (species == "streptococcus" || species == "streptococcus pyogenes") {
```

Comment:
I suggest simplifying the if-clause a bit:
```groovy
if (species in ["streptococcus", "streptococcus pyogenes"]) {
```
(Checked with [nf console](https://www.nextflow.io/docs/latest/reference/cli.html#console) that it works)